src/utils/filplus-edition.ts

```diff
@@ -1,25 +1,44 @@
-export type FilPlusEdition = {
+import { DateTime } from 'luxon';
+
+export type FilplusEditionConfig = {
   id: number;
   start?: number;
   end?: number;
   lowReplicaThreshold: number;
   highReplicaThreshold: number;
 };
 
+export type FilPlusEdition = {
+  startDate: Date;
```

Comment:
I dont like having redundant `start` and `startDate` properties here. Same for `end` and `endDate`. I think we can stick with `Date` types and use this only